docs/userguide/intro.md

```diff
@@ -64,6 +65,25 @@ learning how to manage data, volumes and mounts inside our containers.
 
 Go to [Managing Data in Containers](../tutorials/dockervolumes.md).
 
+## Managing Metadata (Labels) for Docker Objects
```

Comment:
s/Managing Metadata (Labels) for Docker Objects/Managing metadata (labels) for Docker objects/